README.md

```diff
@@ -11,6 +11,10 @@ Golang version 1.12 or higher
 
 # Installation
 
+## Precompiled Binary
+See the [Latest Release](https://github.com/gomicro/flow/releases/latest) page for a download link to the binary compiled for your system.
+
+## From Source
```

Comment:
It's more clear to put the Go compiler version inside this "From Source" section. Otherwise the README can be interpreted as "you need Go installed to use the compiled binary" which isn't true. 😄